src/Microsoft.ML.Data/Transforms/TransformsExtensionsCatalog.cs

```diff
@@ -48,4 +48,48 @@ public static ColumnConcatenatingEstimator Concatenate(this TransformsCatalog ca
             => new ColumnConcatenatingEstimator(CatalogUtils.GetEnvironment(catalog), outputColumn, inputColumns);
 
     }
+
+    /// <summary>
+    /// Extensions for ColumnSelectingEstimator.
+    /// </summary>
+    public static class ColumnSelectingCatalog
+    {
+        /// <summary>
+        /// KeepColumns is used to select a list of columns that the user wants to keep on a given an input. Any column not specified
+        /// will be dropped from the output output schema.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="columnsToKeep">The array of column names to keep.</param>
+        public static ColumnSelectingEstimator KeepColumns(this TransformsCatalog catalog, params string[] columnsToKeep)
+        {
+            return new ColumnSelectingEstimator(CatalogUtils.GetEnvironment(catalog), columnsToKeep);
+        }
+
+        /// <summary>
+        /// DropColumns is used to select a list of columns that user wants to drop from a given input. Any column not specified will
+        /// be maintained in the output schema.
+        /// </summary>
+        /// <param name="catalog">The transform's catalog.</param>
+        /// <param name="columnsToDrop">The array of column names to drop.</param>
+        public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalog, params string[] columnsToDrop)
+        {
+            return new ColumnSelectingEstimator(CatalogUtils.GetEnvironment(catalog), null, columnsToDrop);
```

Comment:
>null, columnsToDrop [](start = 86, length = 19)

it would be more readable to use named arguments here and above, b/c ColumnSelectingEstimator takes so many optional arguments.